lib/customer.rb

```diff
@@ -0,0 +1,10 @@
+class Customer
+
+	attr_reader :number
+
+	def initialize(number)
+		@number = number
+	end
+
+
```

Comment:
Set your indentation in Sublime to be 2 spaces. The Sublime pill explains how to do this.